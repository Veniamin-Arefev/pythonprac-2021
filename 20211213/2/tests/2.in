(rand := __import__("random"), rand.seed("Taki eche testik"), [rand.randint(-100, 100) for i in range(65)])[2]