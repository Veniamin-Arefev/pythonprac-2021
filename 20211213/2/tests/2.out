[-100, -97, -94, -93, -85, -83, -81, -77, -75, -75, -73, -66, -62, -60, -58, -56, -53, -47, -42, -35, -30, -30, -25, -25, -22, -17, -16, -14, -14, -11, -11, -9, 1, 4, 11, 12, 14, 17, 17, 21, 22, 23, 28, 29, 30, 35, 35, 35, 35, 36, 37, 37, 46, 47, 50, 51, 59, 60, 68, 72, 73, 73, 84, 84, 92]
